etc/deploy/aws.sh

```diff
@@ -97,6 +97,7 @@ create_s3_bucket() {
     #
     # TODO(msteffen) pass $NAME in as an argument--relying on the fact that it
     # was already exported by deploy_k8s_on_aws is brittle and hard to read
+    # shellcheck disable=SC2016
```

Comment:
This was a false positive, on line 104 it thinks the template variables are shell variables but they're not.